internal/catalogmetadata/client/client.go

```diff
@@ -111,6 +115,24 @@ func (c *Client) PopulateCache(ctx context.Context, catalog *catalogd.ClusterCat
 	return c.cache.Put(catalog.Name, catalog.Status.ResolvedSource.Image.Ref, resp.Body, nil)
 }
 
+func logX509Error(err error, l logr.Logger) {
+	var cvErr *tls.CertificateVerificationError
+	if errors.As(err, &cvErr) {
+		n := 1
+		for _, cert := range cvErr.UnverifiedCertificates {
+			l.Error(err, "Unverified Cert", "n", n, "subject", cert.Subject, "issuer", cert.Issuer, "DNSNames", cert.DNSNames, "serial", cert.SerialNumber)
+			n = n + 1
+		}
+		return
+	}
+
+	var uaErr *x509.UnknownAuthorityError
```

Comment:
Looking at your log output, I definitely see the `UnknownAuthorityError` embedded in the `CertificateVerificationError`.

In order to be even more useful, it would be good to show extra details of both if the chain is `tls.CertificateVerificationError -> x509.UnknownAuthorityError`

Reply:
We determined that the UnknownAuthorityError did not add anything of value.